extras/analyze-intermittency-reports.sh

```
@@ -0,0 +1,193 @@
+		WORKFLOW="$2"
+		shift 2
+		;;
+	*)
+		echo "Usage: extras/analyze-retry-reports.sh [--runs N] [--workflow NAME]"
+		exit 1
+		;;
+	esac
+done
+
+WORK_DIR=$(mktemp -d)
+trap "rm -rf $WORK_DIR" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)